README.md

```diff
@@ -92,8 +93,8 @@ All of the following have been installed and pre-configured:
 
 ### Base system
 
-- Rails 5.2.1
-- Ruby 2.5.1
+- Rails 6
+- Ruby 2.6.5
```

Comment:
Should we just say Rails and Ruby instead of trying to keep this updated with version numbers?

Reply:
Agreed, that info lives in other files. That's actually the reason I wrote Rails 6, not Rails 6.0.2 😄